filetypes.md

```diff
@@ -0,0 +1,56 @@
+| Mime type                                                                 | Hoover Filetype | Analyzers      |
+| ---                                                                       | ---             | ---            |
+| application/x-7z-compressed                                               | archive         | archives       |
+| application/zip                                                           | archive         | archives       |
+| application/rar                                                           | archive         | archives       |
+| application/x-rar                                                         | archive         | archives       |
+| application/x-zip                                                         | archive         | archives       |
+| application/x-gzip                                                        | archive         | archives       |
+| application/gzip                                                          | archive         | archives       |
+| application/x-bzip2                                                       | archive         | archives       |
+| application/x-tar                                                         | archive         | archives       |
+| application/x-hoover-pst                                                  | email-archive   | archives       |
+| application/mbox                                                          | -               | archives       |
```

Comment:
let's map this to `email-archive` in the code and here too. Does mapping it in the code change any logic? if it does, we want to refactor it later so the logic doesn't depend on these values...